doc/examples/read_experiment_and_data/example_experiment_data.py

```diff
@@ -116,18 +118,16 @@ def run(self):
             axis = matrix.col(goniometer.get_rotation_axis_datum())
             F = matrix.sqr(goniometer.get_fixed_rotation())
             S = matrix.sqr(goniometer.get_setting_rotation())
-            R = S * axis.axis_and_angle_as_r3_rotation_matrix(angle, deg=True) * F
+            R = (
+                S * axis.axis_and_angle_as_r3_rotation_matrix(angle, deg=True) * F
+            )  # noqa F841
         else:
-            R = matrix.sqr((1, 0, 0, 0, 1, 0, 0, 0, 1))
+            R = matrix.sqr((1, 0, 0, 0, 1, 0, 0, 0, 1))  # noqa F841
```

Comment:
I guess all these `# noqa: F841` are because it's an example file for how to process this; I'd suggest instead of doing noqa, doing something like adding a `print` of the values to use them - this would be demonstrative of usage. There _are_ places in the code where things are unfortunately called in order for their side-effects, so an example probably shouldn't give that impression.

Reply:
Yep, sure, I'll do that.  I wasn't very clear on what this example file was trying to show, so I was really just trying to muffle all complaints from it with minimum effort.

Reply:
:+1: